components/Recipe/List-View/Card/index.tsx

```diff
@@ -0,0 +1,96 @@
+/* eslint-disable react/prop-types */
+import React from 'react'
+import Link from 'next/link'
+import styles from './Card.module.css'
+import {Character} from '@lib/types'
+import explosion from 'public/images/Extra/explosion.png'
+import Image from 'next/image'
+
+export interface CardProps {
+  /**
+   * Has this recipe been unlocked?
+   */
+  unlocked?: boolean
+  /**
+   * Recipe title
+   */
+  label: string
+  /**
+   * Image to show
+   */
+  image: string
+  /**
+   * Alt text for image
+   */
+  text: string
+  /**
+   * Color of Card
+   */
+  color: 'Primary' | 'Orange' | 'Teal' | 'Blue'
+  /**
+   * Optional click handler
+   */
+  onClick?: () => void
+  /**
+   * Slug part of the recipe url
+   */
+  slug: string
+  /**
+   * Character for the recipe
+   */
+  character?: Character
+}
+
+const getClassesFromColor = (color: string, unlocked: boolean) => {
```

Comment:
Could add return type to be uber precise `(color: string, unlocked: boolean): string => ...`